### ISSUE TITLE
Error loading listview

### ISSUE THREAD

**Issue by skartknet** [TimGeyssens/UIOMatic#160]
When loading the listview I get an error 'ui-o-matic list state restore failed => TypeError: Cannot read property 'itemsPerPage' of null' on the console.

The error comes from the list.controller.js:99 and it looks like it's trying to get an object from localStorage when it hasn't been saved yet. If I swap the logic on list.controller.js:56 to:

```
if (firstLoad) {
        saveState();
} else {
        loadState();
}
```

it seems to work, or at least I'm not getting the error anymore.

**Comment by TimGeyssens** [comment]
Ah ok, will check thanks for reporting!

**Comment by TimGeyssens** [comment]
Fixed, version 3.0.15 is building now!
